engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/telemetry/TelemetryReporterTest.java

```diff
@@ -1015,6 +1016,50 @@ public void shouldSendDataWithWebapps() {
         .withHeader("Content-Type",  equalTo("application/json")));
   }
 
+  @Test
+  public void shouldSendDataWithDataCollectionTimeFrame() {
+    // given default telemetry data
+    managementService.toggleTelemetry(true);
+
+    stubFor(post(urlEqualTo(TELEMETRY_ENDPOINT_PATH))
+        .willReturn(aResponse()
+            .withStatus(HttpURLConnection.HTTP_ACCEPTED)));
+
+    // we need to create these objects before the reporter fires because the data collection time frame will be reset
+    TelemetryDataImpl expectedData = getTelemetryDataIgnoreCollectionTimeFrame();
```

Comment:
As discussed the test is ignoring the timeframe at the moment
```suggestion
    TelemetryDataImpl expectedData = configuration.getTelemetryData();
```

Reply:
The above is incorrect. The right approach is #extendData(). Then the assert for any date if possible.